src/helpers/transformUniqueAssetTraitsForPresentation.ts

```diff
@@ -0,0 +1,55 @@
+import { format } from 'date-fns';
+import { UniqueAssetTrait } from '../entities/uniqueAssets';
+
+type MappedTrait = UniqueAssetTrait & {
+  originalValue: string | number;
+  keepLowerCase?: boolean;
+  disableMenu?: boolean;
+};
+
+/**
+ * Mapper function that converts various NFT trait values according to their
+ * data type and other qualities. Like if it's a link it is shortening it.
+ */
+export default function transformUniqueAssetTraitsForPresentation(
+  trait: UniqueAssetTrait
+): MappedTrait {
+  const { display_type, value } = trait;
+
+  if (display_type === 'date') {
+    // the value is in seconds, formatted like Jan 29th, 2022
```

Comment:
actually, in milisecs I think. I can be non-integer

Reply:
I will expand, the non-decimal part is in seconds, the decimal part is actually the millisecond part

Reply:
When I rendered it thinking it's milliseconds it made the year be like 1970